website/server/middlewares/notFound.js

```diff
@@ -14,6 +14,7 @@ const TOP_LEVEL_ROUTES = [
   '/export',
   '/email',
   '/qr-code',
+  '/logout'
```

Comment:
why was this added here? This way if you go to habitica.com/logout the client won't be loaded, maybe you wanted to add /logout-server here?